process/peer/process.go

```diff
@@ -1195,5 +1196,6 @@ func (vs *validatorStatistics) LastFinalizedRootHash() []byte {
 // EpochConfirmed is called whenever a new epoch is confirmed
 func (vs *validatorStatistics) EpochConfirmed(epoch uint32) {
 	vs.flagJailedEnabled.Toggle(epoch >= vs.jailedEnableEpoch)
+	vs.flagInactiveToJailCorrection.Toggle(epoch == vs.jailedEnableEpoch)
```

Comment:
Should not be >= ?

Reply:
no - this should be done only once. In the epoch when the activation happens. we have a case with the old code - before flag activation - peer.UnStakedEpoch == maxuint32 && peer.List == Inactive means that it is jailed..

Reply:
ok